themes/bootstrap3/templates/eds/advanced.phtml

```diff
@@ -7,44 +7,30 @@
   ob_start();
 ?>
 <div id="new_group_template">
-  <div class="group well clearfix">
+  <div class="adv-group eds-adv">
     <input type="hidden" name="join" value="AND"/>
-    <input type="hidden" name="bool0[]" value="AND"/>
```

Comment:
Another serious problem here: I'm not sure why you removed the bool0[] value here, but I notice that the boolean drop-downs in the EDS advanced search screen are broken in this branch. If you try a search like "term x" NOT "term y," the NOT reverts to an AND when you submit the search. The booleans work correctly in master.